src/bindings.c

```diff
@@ -973,5 +1003,21 @@ void *lxcfs_fuse_init(struct fuse_conn_info *conn, void *data)
 	can_use_sys_cpu = true;
 #endif
 	has_versioned_opts = true;
-	return fc ? fc->private_data : NULL;
+        return fc ? fc->private_data : NULL;
+}
+
+bool set_runtime_path(const char* new_path)
+{
+        int pathlen;
+
+        if (new_path && strlen(new_path) < PATH_MAX) {
+                pathlen = strlen(new_path);
+                memcpy(runtime_path, new_path, pathlen);
+                runtime_path[pathlen] = '\0';
```

Comment:
Why not strcpy here?

Reply:
For copying the middle parts of strings like in your link it makes sense, but here all of this math can be replaced by a simple strcpy(), IIUC.

Reply:
moved to strcpy